lib/mini_profiler.rb

```diff
@@ -262,7 +195,15 @@ def call(env)
       end
 
       # handle all /mini-profiler requests here
-      return client_settings.handle_cookie(serve_html(env)) if path.start_with? @config.base_url_path
+      if path.start_with? @config.base_url_path
+        file_name = path.sub(@config.base_url_path, '')
+
+        return serve_results(env) if file_name.eql?('results')
```

Comment:
`case`? 🤔

Reply:
Yeah, good call. This was the code in the original method, but can change it while we are here